crates/nu-command/src/filters/get.rs

```diff
@@ -114,6 +92,14 @@ impl Command for Get {
                     span: Span::test_data(),
                 }),
             },
+            Example {
+                description: "Get a column from a table, and return null for any rows where that column does not exist",
+                example: "[{A: A0} {}] | get ?.A",
```

Comment:
Consider adding an example for `get .A` as well, with the text indicating that the dot is strictly optional. Such an example would provide needed clarity as to what the `?.` syntax "means". (Currently, it sorta looks like the ? is another cell name, when actually it's an extension of the separator between names.)

Reply:
First attempt:
![image](https://user-images.githubusercontent.com/26268125/208596632-306b472e-e0bb-43db-91d6-17e392097f26.png)

I feel like that could still be a little more clear. It's late here, will sleep on it.